### ISSUE TITLE
Internationalize phone numbers

### ISSUE THREAD

**Issue by johnnyshields** [fatfreecrm/fat_free_crm#701]
- All phone inputs should use: https://github.com/jackocnr/intl-tel-input
- Phone numbers should be stored in and validated as E.164 format, e.g. "+12061234567" for a US phone number.
- Existing phone data which doesn't start with a plus should be displayed as is, and the user should see a message "please migrate your phone data to E.164 format"

**Comment by CloCkWeRX** [comment]
I like the control, but that's going to be a lot of data migration for a lot of people, some who would only do business within their own country.
When this lands, we'll want a lot of data migration helpers/rake tasks/etc to make it as painless as possible

**Comment by steveyken** [comment]
Agree. Could we make it an optional extra that can be turned on? I can see
it being very useful for some but not all.

**Comment by johnnyshields** [comment]
@CloCkWeRX @steveyken

**Comment by johnnyshields** [comment]
@CloCkWeRX @steveyken it's a best practice to store all phone numbers in the DB as E.164 format (we can add a seperate "phone_extension" field as extensions aren't included in E.164)

I'm fine add a "international phone" option in the settings but I think it should be set to true for newly created apps when you run "ffcrm:setup"

**Comment by CloCkWeRX** [comment]
> All phone inputs should use: https://github.com/jackocnr/intl-tel-input

Not _quite_ the same, but at least its an input tel with [feat: Format phone numbers as tel: links #1540](https://github.com/fatfreecrm/fat_free_crm/pull/1540) and v0.25.0

**Comment by CloCkWeRX** [comment]
I am for the moment going to call this "done"; though what has been implemented is less complete that what was articulated here. If we're really keen on it, can reopen.
I think the 'click to call' UI behaviour though achieves a very similar goal to the original intent, which bludgeons the phone numbers in a better shape.
